### DISCUSSION TITLE
App Directory Discussion Group

### DISCUSSION THREAD

**Discussion by kriswest** [finos/FDC3#618] [Ideas]
This thread will be used to arrange meetings of and agendas for the FDC3 App Directory discussion group. Please feel free to post any requests for additions to upcoming meeting agendas here.

**Comment by kriswest** [comment]
Poll for next meeting date for the App Directory discussion group (to repeat every 2 weeks - with some minor adjustment for upcoming daylight savings time changes):

(Please react with the relevant emoji to express your preference)

- 🎉 Wednesday 16th March 9am EDT / 1pm GMT
- ❤️ Friday 18th March 9am EDT / 1pm GMT 
- 😄 Monday 21st March 9am EDT / 1pm GMT
- 👀 Friday 25th March 9am EDT / 1pm GMT 
- 👎  Need other options

(Note meetings will move to 9am EDT / 2pm BST for next meeting)

**Reply by kriswest** [reply]
I've received a couple of offline votes for 😄 Monday 21st (+Vinay below), hence, I think that's the current favourite option.

@ggeorgievx can you make then (option added after your initial response)?

**Reply by ggeorgievx** [reply]
Yes I can, but might need to drop earlier. Thank you, @kriswest!

**Reply by kriswest** [reply]
😄 Monday 21st March 9am EDT / 1pm GMT it is!

**Comment by mistryvinay** [comment]
😀
Vinay / Symphony

**Comment by kriswest** [comment]
Here is a rough grouping of the open AppD related issues which we can use to plan out upcoming meetings:
 
## Defining the AppD (discussed at #626)
- #555
  - #570
- #556
  - Note the changes made in #437
- #560
- #558
  - #570
- #561

## Improving the application record (discussed at #657 and #664)
- #561	
- #369
- #622
- #247
- #567
- #550
- #562
- #564

## Improving the AppD API
- #553
- #370
- #311
- #565
- #557
- #566
- #559

## Other
- #312
- #563
